### PR TITLE
Fix the return as an object, not a list, in the /validator

### PR DESCRIPTION
**Pull request by TrustHenry**

### CODE REVIEW

File: src/Stoa.ts [hunk 1]
```diff
@@ -98,18 +98,12 @@ class Stoa {
...
-                    for (const row of rows)
-                    {
-                        let preimage: IPreimage = {distance: row.distance, hash: '0'} as IPreimage;
-                        var validator: ValidatorData | null =
-                            new ValidatorData(row.address, row.enrolled_at, row.stake, preimage);
-                        out_put.push(validator);
-                    }
-                    res.status(200).send(JSON.stringify(out_put));
+                    let preimage: IPreimage = {distance: rows[0].distance, hash: '0'} as IPreimage;
+                    var validator: ValidatorData | null =
```
Comment (MichaelKim20) [inline src/Stoa.ts]:
```suggestion
                    let validator: ValidatorData | null =
```

**Review by MichaelKim20 — CHANGES_REQUESTED** [review]
We'd better use 'let' rather than 'var' for variable declarations in the future. 
This is a reference to this.
English
https://www.typescriptlang.org/docs/handbook/variable-declarations.html
Korean
https://infoscis.github.io/2017/05/14/TypeScript-handbook-variable-declarations/

File: src/Stoa.ts (hunk 1 above)
Comment (TrustHenry) [reply src/Stoa.ts]: That's a good point.

Comment (TrustHenry) [reply src/Stoa.ts]: I apply the suggestion now.

**Review by MichaelKim20 — APPROVED** [review]
LGTM

File: src/Stoa.ts [hunk 2]
```diff
@@ -98,18 +98,12 @@ class Stoa {
                     return;
                 }
 
-                if (rows.length)
+                if (rows.length == 1)
```
Comment (Geod24) [inline src/Stoa.ts:101]: That's a weird change. What happen if we were to get `rows.length == 2` ?

File: src/Stoa.ts [hunk 3]
```diff
@@ -98,18 +98,12 @@ class Stoa {
...
-
-                    for (const row of rows)
-                    {
-                        let preimage: IPreimage = {distance: row.distance, hash: '0'} as IPreimage;
-                        var validator: ValidatorData | null =
-                            new ValidatorData(row.address, row.enrolled_at, row.stake, preimage);
-                        out_put.push(validator);
-                    }
-                    res.status(200).send(JSON.stringify(out_put));
+                    let preimage: IPreimage = {distance: rows[0].distance, hash: '0'} as IPreimage;
```
Comment (Geod24) [inline src/Stoa.ts:103]: Ah that explains it :D

**Comment by MichaelKim20** [comment]
I called the address below to get the following results.

```
http://localhost:3836/validator/GBUVRIIBMHKC4PE6BK7MO2O26U2NJLW4WGGWKLAVLAA2DLFZTBHHKOEK
```

```
{"address":"GBUVRIIBMHKC4PE6BK7MO2O26U2NJLW4WGGWKLAVLAA2DLFZTBHHKOEK","enrolled_at":0,"stake":"0x86f1a6dff3b1f2256d2417b71ecc5511293b224894da5fd75c192965aa1874824ca777ecac678c871e717ad38c295046f4f64130f31750aa967c30c35529944a","preimage":{"distance":0,"hash":"0"}}
```

**Comment by Geod24** [comment]
What about the previous comment ?

File: src/Stoa.ts (hunk 2 above)
Comment (TrustHenry) [reply src/Stoa.ts:101]: This is the function of returning a single validator value.
If `rows.length ==2` return 204 to HTTP response code.

File: src/Stoa.ts (hunk 3 above)
Comment (TrustHenry) [reply src/Stoa.ts:103]: I will open another PR related to the preimage.

File: src/Stoa.ts (hunk 2 above)
Comment (Geod24) [reply src/Stoa.ts:101]: But we look it up by address. We should NEVER have `res.length > 1`.

Comment (Geod24) [reply src/Stoa.ts:101]: Actually scratch that. Currently, it is possible that a validator uses multiple UTXOs to enroll.

Comment (TrustHenry) [reply src/Stoa.ts:101]: Oh, that's right.
I think I overlooked it.
I'll open a new PR.

Comment (TrustHenry) [reply src/Stoa.ts:101]: If so, does the validator of the function return an enrollment of one of the UTXOs?

Comment (Geod24) [reply src/Stoa.ts:101]: I would say all of them, potentially sorted.

**Pull request merged**